### PR TITLE
code with Mistakes

### PR DESCRIPTION
**Pull request by ArtemNemtsov**

### CODE REVIEW

File: kitsune/menu/menu.h [hunk 1]
```diff
@@ -26,13 +26,13 @@ namespace kitsune
         /*! Activate next item */
         void ActivateNext() 
         {
-            // your code here
+            activatedIndex_++;
```
Comment (eaniconer) [inline kitsune/menu/menu.h]: What if `activateIndex` is more or equal size of menu?

File: kitsune/menu/menu.h [hunk 2]
```diff
@@ -26,13 +26,13 @@ namespace kitsune
...
         }
 
         /* Activates previous item. */
         void ActivatePrev() 
         {
-            // your code here
+            activatedIndex_--;
```
Comment (eaniconer) [inline kitsune/menu/menu.h]: What if `activateIndex` become negative number, i.e. -2, -3, -4

File: kitsune/menu/main.cpp [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
 {
-    // your code here
+    menu.GetItems();
 }
 
 int main()
 {
-    MenuList<string> menu;
-    LoadMenuFromFile(menu, "Myfile");
+	MenuList<string> menu;
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: Fix indentation

File: kitsune/menu/main.cpp [hunk 4]
```diff
@@ -1,25 +1,44 @@
...
 }
 
+
+
+
+
 void PrintMenu(const MenuList<string>& menu)
 {
-    // your code here
+    menu.GetItems();
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: You have to print item names in `stdout`:
Something like this:
`for (/*your code here*/) `
     `cout << /*your code here*/ << endl;`

File: kitsune/menu/main.cpp [hunk 5]
```diff
@@ -1,25 +1,44 @@
...
+#include <fstream>
+#include <conio.h>
 
 using namespace std;
 using namespace kitsune;
 
 void LoadMenuFromFile(MenuList<string>& menu, string filename)
 {
-    // your code here
+  ifstream fin("Menu.txt");
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: We want to choose file name in main() function, not here, use `filename`

File: kitsune/menu/main.cpp [hunk 6]
```diff
@@ -9,19 +9,19 @@ using namespace kitsune;
 
 void LoadMenuFromFile(MenuList<string>& menu, string filename)
 {
-  ifstream fif("filename");
-  if (fif.is_open()) 
+    ifstream fif("filename");
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: Use variable `filename`, not string!

File: kitsune/menu/main.cpp [hunk 7]
```diff
@@ -1,25 +1,64 @@
...
+    cfi.dwFontSize.Y = 35;                  // Height
+    cfi.FontFamily = FF_DONTCARE;
+    cfi.FontWeight = FW_NORMAL;
+    std::wcscpy(cfi.FaceName, L"Consolas"); // Choose your font
+    SetCurrentConsoleFontEx(GetStdHandle(STD_OUTPUT_HANDLE), FALSE, &cfi);
+    vector<string> res = menu.GetItems();
+    cout <<string( 10, '\n' );
+    for(int i = 0; i < res.size(); i++)
+    {
+        menu.ActivatedIndex();
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: This method returns activated index. Use it in the following if-condition to choose activated element.

File: kitsune/menu/main.cpp [hunk 8]
```diff
@@ -1,25 +1,64 @@
...
+        if(i==0)//How to be here? i can write every index, but what, if I dont how many items in file? 
+        {
+            cout<<"                                 *" <<res[i] <<"*"<< endl;
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: if you want to print many spaces you can use `string(10, ' ')`. Now you can control amount of them.

File: part1_ConsoleSnake/Безымянный1.cpp [hunk 9]
```diff
@@ -0,0 +1,47 @@
+#include <iostream>
```
Comment (eaniconer) [inline part1_ConsoleSnake/Безымянный1.cpp]: Why do we need this file?

File: kitsune/menu/main.cpp (hunk 8 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: ok.Merci

File: part1_ConsoleSnake/Безымянный1.cpp (hunk 9 above)
Comment (ArtemNemtsov) [reply part1_ConsoleSnake/Безымянный1.cpp]: << iostream uses cin, cout, cerr, and clog objects to transfer information to and from standard input, output, error (without buffering) and errors (with buffering) respectively. >> - this means that using these objects, we we can display the text on the screen.

File: kitsune/menu/main.cpp (hunk 7 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: ok. I deleted this method here

File: kitsune/menu/main.cpp (hunk 5 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: ok. I used it

File: kitsune/menu/main.cpp (hunk 4 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: for(int i = 0; i < res.size(); i++)// ShowFirstTime
    {
        if(i==0 )
        {
            cout <<string( 10, '\n' );
            cout<<string(60, ' ')<<"*" <<res[i] <<"*"<< endl;
        }
        else cout<<string(60, ' ')<<res[i] << endl; 
   }

File: kitsune/menu/main.cpp (hunk 3 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: done

File: kitsune/menu/main.cpp (hunk 7 above)
Comment (eaniconer) [reply kitsune/menu/main.cpp]: You should save value returned by this method. Then use the value in condition statement.

**Comment by eaniconer** [comment]
Wait new request

**Pull request closed without merging**